includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1916,6 +1921,17 @@ public function wc_setup_recommended() {
 						'plugins'     => array( array( 'name' => __( 'MailChimp for WooCommerce', 'woocommerce' ), 'slug' => 'mailchimp-for-woocommerce' ) ),
 					) );
 				endif;
+
+				if ( $this->should_show_facebook() ) :
```

Comment:
Nitpicking; don't think we need to if/endif syntax here. Braces are better in a big block of PHP.

Reply:
Ah yes I see. I think I first got this idea from https://make.wordpress.org/core/handbook/best-practices/coding-standards/html/#indentation but now there's no HTML there anymore, so it's not really the best pattern. Are you ok with keeping this here to match the rest of the code, or would you like me to update that entire function back to braces?